packages/vitepress-plugin-git-changelog/src/client/components/CommitRegularLine.vue

```diff
@@ -13,23 +13,23 @@ const { t } = useI18n()
 </script>
 
 <template>
-  <div class="i-octicon:git-commit-16 m-auto rotate-90 transform opacity-30" />
-  <div flex items-center gap-1>
+  <div class="i-octicon:git-commit-16 m-auto rotate-90 transform op-30" />
+  <div flex gap-1>
     <a :href="props.commit.hash_url || `${props.commit.repo_url}/commit/${props.commit.hash}`" target="_blank">
       <code
         class="text-xs text-$vp-c-brand-1 hover:text-$vp-c-brand-1"
         transition="color ease-in-out"
         duration-200
       >
-        {{ commit.hash.slice(0, 7) }}
+        {{ commit.hash.slice(0, commit.commitHashDigits) }}
       </code>
     </a>
     <span>-</span>
     <span>
       <span class="text-sm <sm:text-xs" v-html="renderCommitMessage(commit.repo_url || 'https://github.com/example/example', commit.message)" />
       <ClientOnly>
-        <span class="text-xs opacity-50" :title="toDate(commit.date_timestamp).toString()">
-          {{ t('committedOn', { props: { date: toDate(commit.date_timestamp).toLocaleDateString() }, omitEmpty: true }) || t('changelog.committedOn', { props: { date: toDate(commit.date_timestamp).toLocaleDateString() } }) }}
+        <span class="text-xs op-50" :title="toDate(commit.date_timestamp).toString()">
+          {{ t('committedOn', { props: { date: toDate(commit.date_timestamp as unknown as Date).toLocaleDateString() }, omitEmpty: true }) || t('changelog.committedOn', { props: { date: toDate(commit.date_timestamp as unknown as Date).toLocaleDateString() } }) }}
```

Comment:
```suggestion
          {{ t('committedOn', { props: { date: toDate(commit.date_timestamp).toLocaleDateString() }, omitEmpty: true }) || t('changelog.committedOn', { props: { date: toDate(commit.date_timestamp).toLocaleDateString() } }) }}
```

`date_timestamp` is a `number`, basically a UNIX timestamp.

Reply:
since I got error `Type argument cannot be inferred from usage` in webstorm, so I add the type assertion, and after that the error disappears

Reply:
Ok, I see. Could you please try to notate it as `as unknown as number`?

Reply:
`as unknown as number` does not work, maybe we could ignore this error by webstorm